source/test/TestApp/PlayFabTestMultiUserInstance.cpp

```diff
@@ -14,6 +14,12 @@ using namespace ClientModels;
 
 namespace PlayFabUnit
 {
+    std::string multiUserInstanceTitleId;
```

Comment:
Any specific reason we are storing this as a global here instead of a member variable?

Reply:
It's not really global since it's only defined in the cpp.
I just wanted a temp variable to carry the titleId from SetTitleInfo to the SetUp function, which is called much later.

---
In reply to: [354069484](https://github.com/PlayFab/XPlatCoreTemplate/pull/59#discussion_r354069484) [](ancestors = 354069484)

Reply:
Alright, I'll fix.

---
In reply to: [354069858](https://github.com/PlayFab/XPlatCoreTemplate/pull/59#discussion_r354069858) [](ancestors = 354069858,354069484)